style.css

```diff
@@ -4,10 +4,19 @@
   box-sizing: border-box;
 }
 
+@import url("https://rsms.me/inter/inter.css");
+html {
+  font-family: "Inter", sans-serif;
+}
+@supports (font-variation-settings: normal) {
+  html {
+    font-family: "Inter var", sans-serif;
+  }
+}
```

Comment:
@xtrahuman I didn't get your point, do you need me to write this
```
 html {
    font-family: "Inter", sans-serif;
  }
```
 instead of this
 ```
 html {
    font-family: "Inter var", sans-serif;
  }
```

Reply:
yes, please dont use this
`html {
   font-family: "Inter var", sans-serif;
 }`